helm-chart/eoapi/templates/services/nginx-doc-server.yaml

```diff
@@ -16,6 +16,7 @@ data:
           <li><a href="/raster" target="_blank" rel="noopener noreferrer">/raster</a></li>
           <li><a href="/vector" target="_blank" rel="noopener noreferrer">/vector</a></li>
           <li><a href="/stac" target="_blank" rel="noopener noreferrer">/stac</a></li>
+          <li><a href="/browser/" target="_blank" rel="noopener noreferrer">/browser</a></li>
```

Comment:
For me the term `browser` seems to general. I would prefer to have this under `/stac/...` somehow, or find a more explicit wording. The boring approach would be `stac-browser`;

A very beautiful would be to integrate it on an endpoint level, like [here](https://github.com/developmentseed/eoapi-devseed/pull/17). But this is a different direction and out-of-scope for this PR/approach.

Reply:
I'd be happy with `stac-browser`.

Reply:
The current ingress configuration use often prefix to define the service where the request is served so I kept /browser until we have a better solution.